assignments/arrays.js

```diff
@@ -63,32 +63,53 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// let lastCar = 0;
+// lastCar = inventory.pop();
+// console.log(`The make and model of the last car is: ${lastCar.car_make} ${lastCar.car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+// let carModels = [];
+// carModels = inventory.sort((a, b) => (a.car_model > b.car_model)? 1: -1);
+// console.log(carModels);
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
-let carYears = [];
-console.log();
+// let carYears = [];
+// carYears = inventory.map( (element) => element.car_year);
+// console.log(carYears);
```

Comment:
you you only have one parameter being passed in the arrow function you dont need the parenthesis around it, so the map can look like this... inventory.map( element => element.car_year)